charmander/app.py

```diff
@@ -0,0 +1,62 @@
+"""
+Create the application.
+
+"""
+from microcosm.api import create_object_graph
+from microcosm.loaders import load_each, load_from_environ, load_from_json_file
+from microcosm.loaders.compose import load_config_and_secrets
+from microcosm_secretsmanager.loaders.conventions import load_from_secretsmanager
+
+import charmander.postgres  # noqa
+import charmander.routes.example.controller  # noqa
+import charmander.routes.example.crud   # noqa
+import charmander.stores.example_store  # noqa
+from charmander.config import load_default_config
+
+
+def create_app(debug=False, testing=False, model_only=False):
```

Comment:
Both services and daemons have an `app` which is basically a `microcosm.graph` instance which encapsulates all of the apps components.